apps/pie-docs/src/components/spinner/motion/motion.md

```diff
@@ -0,0 +1,34 @@
+---
+eleventyNavigation:
+    key: Motion
+    parent: 'Spinner'
+    order: 2
+---
+
+## Example
+
+Illustrates the animations that happen in the button component, on hover, active and loading states. You can see an animated example of this component [here](https://drive.google.com/drive/folders/1uz-RLSiNUkjDccpI89puM1cp_whpGobX).
```

Comment:
The video linked doesn't show any loading state, just hover and active state 👀 is that expected?
also the video will be restricted for public. 

maybe a better idea is to upload it to Youtube and use the youtube short-code to display it like we do here? https://pie.design/patterns/overlay-patterns/ios-guidance/

Reply:
let's ask the team in the standup 🤝